internal/cmd/cmd.go

```diff
@@ -123,12 +123,11 @@ zed permission check --explain document:firstdoc writer user:emilia
 
 	relCmd := commands.RegisterRelationshipCmd(rootCmd)
 
-	commands.RegisterWatchCmd(rootCmd)
 	commands.RegisterWatchRelationshipCmd(relCmd)
 
 	schemaCmd := commands.RegisterSchemaCmd(rootCmd)
-	schemaCompileCmd := registerAdditionalSchemaCmds(schemaCmd)
-	registerPreviewCmd(rootCmd, schemaCompileCmd)
```

Comment:
We no longer pass out the `schemaCompileCmd` as it's no longer needed as an arg to the preview command.